proxy/proxy.go

```diff
@@ -1070,6 +1070,25 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 	p.sendError(ctx, id, code)
 }
 
+func shouldLog(statusCode int, filter al.AccessLogFilter) bool {
+	if len(filter.Prefixes) == 0 {
+		return filter.Enable
```

Comment:
Zero value of bool is false, so the default would be not to log

Reply:
to me this seems fine, because below, before calling shouldLog, the argument is initialized with !p.accessLogDisabled

Reply:
i think this is fine, because shouldLog is called with an initialized Enable value, by default taking !p.accessLogDisabled

Reply:
Yep. The value of `Enable` is set in `createFilter`.

Reply:
Ok I am fine with it